data/steam

```diff
@@ -3,11 +3,13 @@ s.team
 steam-chat.com
 steamcommunity.com
 steamcontent.com
+steamdeck.com
 steamgames.com
 steampowered.com
 steamstatic.com
 steamusercontent.com
 underlords.com
+valve.net
```

Comment:
I double checked this domain, and only find one with ssl subdomain which will guide to a login form. The SSL certification shows it has some relationship with valve corp, however this is not kind of strong evidence I belive.